### PR TITLE
Open locator popup with shortcut

### PR DESCRIPTION
**Pull request by meretp**
### Summary of changes

With this PR the previously introduced popup to locate signals can be openend via the shortcut Ctrl+L or Cmd+L or via the Top bar. I also added a new icon for this functionality.

### Context and reason for change

We want to imlement a new feature to be able to locate signals based on criticality and certain licenses. 

### How can the changes be tested
Run the unittests.
Check out the PR and start OpossumUI. If you don't open a file the shortcut Ctrl+L or Cmd+L and will do nothing. But you will see the option "Locate Signals" in the top bar menu as in the screenshot below.
image

Next, open a file and use Ctrl+L or Cmd+L and observe that the popup shows up as in the screenshot below. 
image

Solves #1947.

### CODE REVIEW

File: src/ElectronBackend/main/menu.ts [hunk 1]
```diff
@@ -243,6 +243,26 @@ export function createMenu(mainWindow: BrowserWindow): Menu {
...
+        {
+          icon: nativeTheme?.shouldUseDarkColors
+            ? path.join(
+                getBasePathOfAssets(),
+                'icons/location-searching-white.png',
+              )
+            : path.join(
+                getBasePathOfAssets(),
+                'icons/location-searching-black.png',
+              ),
```
Comment (benedikt-richter) [inline src/ElectronBackend/main/menu.ts:255]: It seems this code is repeated a lot. Might be nice to rafactor it such that it takes two file names and returns the icon that fits the theme.

**Comment by benedikt-richter** [comment]
The commit message should contain the issue number and a description what was done. Would be nice if you could add that.

File: src/ElectronBackend/main/menu.ts (hunk 1 above)
Comment (meretp) [reply src/ElectronBackend/main/menu.ts]: Good idea, I extracted a method for this.

**Review by benedikt-richter — APPROVED** [review]
👍

**Pull request merged**